iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/HvdcLineAdderAdapter.java

```diff
@@ -130,6 +133,12 @@ public HvdcLineAdder setEnsureIdUnicity(final boolean ensureIdUnicity) {
         return this;
     }
 
+    @Override
+    public HvdcLineAdder setFictitious(final boolean fictitious) {
+        this.fictitious = fictitious;
+        return this;
+    }
```

Comment:
Do you confirm that HvdcLineAdderAdapter doesn’t inherit from a class which is responsible of the fictitious attribute

Reply:
`HvdcLineAdderAdapter` does not inherit from `AbstractIdentifiableAdderAdapter`
To be discussed:
I can create a new class which is responsible of the id, name, ensureIdUnicity, fictitious & MergingViewIndex
`HvdcLineAdderAdapter` & `LineAdderAdapter` could inherit from it